doc/memory_tune.md

```diff
@@ -30,3 +30,5 @@ Normally there are some methods to tune the memory usage:
 - set the app heap size with `wasm_runtime_instantiate`
 - use `nostdlib` mode, add `-Wl,--strip-all`: refer to [How to reduce the footprint](./build_wasm_app.md#2-how-to-reduce-the-footprint) of building wasm app for more details
 - use XIP mode, refer to [WAMR XIP (Execution In Place) feature introduction](./xip.md) for more details
+- when using the Wasm C API in fast interpreter or AOT mode, set `clone_wasm_binary=false` in `LoadArgs` and free the wasm binary buffer (with `wasm_byte_vec_delete`) after module loading; `wasm_runtime_is_underlying_binary_freeable` can be queried to check if the wasm binary buffer can be safely freed
```

Comment:
when you say "module loading", what exacty is it?
wasm_module_new_ex?

Reply:
`wasm_module_new(_ex)` and `wasm_runtime_load(_ex)`

Reply:
in the description https://github.com/bytecodealliance/wasm-micro-runtime/pull/3389#issue-2277885176 you said "after module instantiation", which makes me think `wasm_instance_new`. can you update the description?